Formula/libomxil-bellagio.rb

```diff
@@ -27,11 +27,11 @@ def install
     # Be explicit about the configure flags
     args << "--enable-static=#{build.with?("static") ? "yes" : "no"}"
 
-    system "autoreconf", "-fiv"
+    #system "autoreconf", "-fiv"
     system "./configure", *args
     ENV.deparallelize
-    system "make"
+    system "CFLAGS='' make"
```

Comment:
@sjackman would `ENV.delete "CFLAGS"` be better suited for this?

Reply:
Thank you. I will try.